### PR TITLE
More info on sharing

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,18 +4,51 @@
 
 R weekly provides weekly updates from the R community. You are welcome to contribute as long as you follow [our code of conduct](CODE_OF_CONDUCT.md) and [our contributing guide](CONTRIBUTING.md).
 
-## How to contribute by using this repo
+## How to have (my) content featured in R Weekly?
```
Comment (jonocarroll) [inline README.md]: 'featured' may imply highlight. What about "submit a post to R Weekly"?

**Review by jonocarroll — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (maelle) [reply README.md]: I see what you mean but your phrase doesn't apply to RSS feeds, does it?

How about "How to have (my) content appear in R Weekly"?

Comment (maelle) [reply README.md]: or "shared by R Weekly"?

Comment (jonocarroll) [reply README.md]: "considered for R Weekly"?

**Comment by maelle** [comment]
I also updated the volunteering info that was slightly out-of-date.

**Comment by maelle** [comment]
@qinwf shouldn't the README indicate how to support R Weekly? If so could you please add a sentence to my branch?

**Review by qinwf — DISMISSED** [review]

**Comment by qinwf** [comment]
I add a link to the draft.

https://github.com/rweekly/rweekly.org/pull/703/commits/7bb7cf742f1951c520f1454e40c460a99d22ade0

**Pull request merged**